samples/charts/financial-chart/scrollbars/App.razor

```diff
@@ -0,0 +1,43 @@
+
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="container vertical">
+    <IgbFinancialChart
+        @ref="Chart"
+        Width="100%"
+        Height="100%"
+        ChartTitle="S&P 500"
+        YAxisTitle="Financial Prices"
+        ChartType="FinancialChartType.Candle"
+        DataSource="Data"
+        ZoomSliderType="FinancialChartZoomSliderType.None"
+        IsToolbarVisible="false"
+        IsVerticalZoomEnabled="true"
+        IsHorizontalZoomEnabled="true"
+        HorizontalViewScrollbarMode=SeriesViewerScrollbarMode.Persistent
+        VerticalViewScrollbarMode=SeriesViewerScrollbarMode.Persistent/>
```

Comment:
can you create this sample into xplat-examples repo and add combo-boxes to change scrollbar properties?

Reply:
sure, but this pr has to be merged because igEditor doesn't support blazor.